src/Element.js

```diff
@@ -11,6 +11,12 @@ var querySelectorAll = require('query-selector')
 function styleCamelCase (name) {
   var camel = camelCase(name)
 
+  // Detect if the style property is already camelCased
+  // To not convert Webkit*, Moz* and O* to lowercase
+  if (camel.slice(1) === name.slice(1)) {
```

Comment:
Can just use `camel[0]` and `name[0]`, not sure which is the better approach, but it's more succinct.